pkg/controllers/execution/execution_controller.go

```diff
@@ -113,18 +113,19 @@ func (c *Controller) Reconcile(ctx context.Context, req controllerruntime.Reques
 	}
 
 	if err := c.updateWorkDispatchingConditionIfNeeded(ctx, work); err != nil {
-		klog.Errorf("Failed to update work condition type %s. err is %v", workv1alpha1.WorkDispatching, err)
+		klog.ErrorS(err, "Failed to update work condition type", "type", workv1alpha1.WorkDispatching)
 		return controllerruntime.Result{}, err
 	}
 
 	if util.IsWorkSuspendDispatching(work) {
-		klog.V(4).Infof("Skip syncing work(%s/%s) for cluster(%s) as work dispatch is suspended.", work.Namespace, work.Name, cluster.Name)
+		klog.V(4).InfoS("Skip syncing work for cluster as work dispatch is suspended.", "work", work.Namespace+"/"+work.Name, "cluster", cluster.Name)
```

Comment:
Shall we just use one pattern for logging work? like
```suggestion
		klog.V(4).InfoS("Skip syncing work for cluster as work dispatch is suspended.", "namespace", work.Namespace, "name", work.Name, "cluster", cluster.Name)
```